Test/allocated1/dafny0/OpaqueFunctions.dfy.expect

```diff
@@ -132,4 +132,4 @@ OpaqueFunctions.dfy(229,19): Error: assertion violation
 Execution trace:
     (0,0): anon0
 
-Dafny program verifier finished with 26 verified, 31 errors
+Dafny program verifier finished with 27 verified, 31 errors
```

Comment:
AFAICT there shouldn't be any additional proof obligations if there aren't any default arguments declared, so what's going on here?

Reply:
Very astute observation. This was a consequence of the bug fix https://github.com/dafny-lang/dafny/pull/1245/commits/d685681a35d80621c7b40ed1642969b7880aa2ec that I did along the way and didn't bother separating into its own PR.